plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/LibraryClasspathContainerSerializer.java

```diff
@@ -130,6 +130,7 @@ public void saveLibraryIds(IJavaProject javaProject, List<String> libraryIds)
     }
   }
 
+  // todo containerPath argument is not used
```

Comment:
Oh! That's definitely not right: the containerPath should be the save location — basically the container ID.

Reply:
File a bug?

Reply:
OK, we can look into that further down the road. I want to get a release cut first.